autofaiss/external/metadata.py

```diff
@@ -258,13 +262,12 @@ def _get_ivf_training_memory_usage_in_bytes(self):
         """Get IVF memory estimation in bytes."""
         return 4.0 * self.params["ncentroids"] * self.dim_vector
 
-    def _get_hndw_training_memory_usage_in_bytes(self, nb_vectors):
-        """Get HNSW memory estimation in bytes."""
-        return nb_vectors * self.params["M_HNSW"] * 2 * 4
-
-    def _get_ivf_hndw_training_memory_usage_in_bytes(self):
+    def _get_ivf_hnsw_training_memory_usage_in_bytes(self):
         """Get HNSW followed by IVF memory estimation in bytes."""
-        return self._get_hndw_training_memory_usage_in_bytes(nb_vectors=self.params["ncentroids"])
+
+        hnsw_graph_in_bytes = self.params["ncentroids"] * self.params["M_HNSW"] * 2 * 4
```

Comment:
hnsw combined with ivf uses an hnsw index at training time (to store the centroids)